NOAA-EMC/EVS
# global_ens: add restart capability to atmos grid2grid and precip stats jobs

## Description of Changes
This PR adds restart capability to atmos grid2grid and precip stats jobs for the global_ens component. This PR addresses Issue #532.

## Developer Questions and Checklist
* Is this a high priority PR? If so, why and is there a date it needs to be merged by? Yes.
* Do you have any planned upcoming annual leave/PTO? Yes, 11/27-29
* Are there any changes needed for when the jobs are supposed to run? No.
  

## Testing Instructions
(1) Set up jobs
a. Symlink the EVS_fix directory locally as "fix".
b. Copy the exec directory from EVS prod package:
`cp -r /lfs/h1/ops/prod/packages/evs.v1.0.13/exec $HOMEevs`

c. In the driver scripts, edit the following environment variables:

HOMEevs - set to your test EVS directory
COMIN - set to /lfs/h2/emc/vpppg/noscrub/emc.vpppg/${NET}/$evs_ver_2d
COMOUT - set to your test output directory

(2) Run jobs
Run the following jobs in EVS/dev/drivers/scripts/stats/global_ens:

qsub jevs_global_ens_cmce_atmos_grid2grid_stats.sh
qsub jevs_global_ens_ecme_atmos_grid2grid_stats.sh
qsub jevs_global_ens_gefs_atmos_grid2grid_stats.sh
qsub jevs_global_ens_naefs_atmos_grid2grid_stats.sh
qsub jevs_global_ens_cmce_atmos_precip_stats.sh
qsub jevs_global_ens_ecme_atmos_precip_stats.sh
qsub jevs_global_ens_gefs_atmos_precip_stats.sh
qsub jevs_global_ens_naefs_atmos_precip_stats.sh

Log files should be checked for free of errors.

(3) Test restart capability
After a successful full run, save the final stat file for comparison. Run the job again and randomly stop the job using qdel. Then, resubmit the job using qsub. The resubmitted job should run shorter than the full run, and the final stat file from the resubmitted job should be the same from the full run. Log file from the resubmitted job should also be free of errors.
